docker/tidb-sysbench/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM alpine:3.9
```

Comment:
Do we need to make this a separate image?

Reply:
yes, because we patch sysbench. If we can get our patch upstream it won't really be needed.